__tests__/app.test.js

```diff
@@ -170,6 +169,62 @@ describe("app", () => {
       });
   });
 
+  describe("POST /api/reviews/:review_id/comments", () => {
+    it("201: responds with posted comment", () => {
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send({
+          username: "philippaclaire9",
+          body: "I am Groot",
+        })
+        .expect(201)
+        .then(({ body: { comment } }) => {
+          expect(comment).toMatchObject({
+            comment_id: 7,
+            body: "I am Groot",
+            review_id: 1,
+            author: "philippaclaire9",
+            votes: 0,
+            created_at: expect.any(String),
+          });
+        });
+    });
```

Comment:
Lets also add another test (400) for if given invalid ID such as reviews/monkey/comments
Another test for (404) is the username does not exist (this should throw PSQL error due to constraints being set that username needs to be in usertable)  
Another 404 test for path such as reviews/1000000000/comments for a post